Makefile

```diff
@@ -0,0 +1,13 @@
+all:
+
+clean:
+
```

Comment:
why `clean:` when it doesn't do anything? If a user wanted to run `make clean`, they would get `make: *** No rule to make target 'clean'.  Stop.` without this, which is appropriate.

Reply:
With clean you will get `make: Nothing to be done for 'clean'.`. As `clean` is a common target I would rather know that the developer knows that there is nothing to clean. Otherwise I will guess if missed out `clean` all together.